### DISCUSSION TITLE
how i caching column visibility status?

### DISCUSSION THREAD

**Discussion by mckaragulle** [Power-Components/livewire-powergrid#1610] [Q&A] [answered]
When a user hides a column in the users table, the column should remain hidden when they come back to the page or refresh the page. How can I do that?

**Comment by fantismic** [comment, marked as answer]
Come back and refresh may be two different things.
You can use [Persist](https://livewire-powergrid.com/table-component/component-configuration.html#persist) to save in cache/session the table config.

**Reply by mckaragulle** [reply]
Thank you, i will try tomorrow.
